src/com/google/enterprise/adaptor/DocumentHandler.java

```diff
@@ -687,6 +690,12 @@ public OutputStream getOutputStream() throws IOException {
             ex.getResponseBody()));
         os = countingOs;
       }
+      if (null != contentTransformer) {
```

Comment:
Actually, I was thinking we might want to create multiple content transforms
per request in a transform pipeline like the metadata transforms.
—
Brett M. Johnson

On Jul 17, 2015, at 3:58 PM, Eric Anderson <notifications@github.com> wrote:

> In src/com/google/enterprise/adaptor/DocumentHandler.java:
> 
> > @@ -687,6 +690,12 @@ public OutputStream getOutputStream() throws IOException {
> >              ex.getResponseBody()));
> >          os = countingOs;
> >        }
> > +      if (null != contentTransformer) {
> This does have some threading issues, as we will be handling multiple requests at the same time. We're going to need a way to create multiple of the content transformers. Probably create a new one for each request.
> 
> —
> Reply to this email directly or view it on GitHub.
>